packages/app/src/debug/utils/debugArtifacts.ts

```diff
@@ -1,23 +1,28 @@
 import type { CloudRunInstance } from '@packages/data-context/src/gen/graphcache-config.gen'
+import { getUrlWithParams } from '@packages/frontend-shared/src/utils/getUrlWithParams'
 import type { useI18n } from '@cy/i18n'
 
 export type ArtifactType = 'TERMINAL_LOG' | 'IMAGE_SCREENSHOT' | 'PLAY'
 
 export type DebugArtifact = { icon: ArtifactType, text: string, url: string }
 
+const formatUrl = (url: string, campaign: string): string => {
+  return getUrlWithParams({ url, params: { utm_medium: 'Debug Tag', utm_campaign: campaign } })
```

Comment:
Should this be `Debug Tab`?

Reply:
Yes, moved to constant and refactor usages of this string